.prettierrc

```diff
@@ -1,6 +1,13 @@
 {
-  "semi": true,
-  "trailingComma": "all",
+  "arrowParens": "avoid",
+  "bracketSameLine": true,
```

Comment:
Could we consider keeping "bracketSameLine" as false to maintain better code readability?

Reply:
Sure